internal/terraform/node_resource_import.go

```diff
@@ -103,12 +103,9 @@ func (n *graphNodeImportState) Execute(ctx EvalContext, op walkOperation) (diags
 	}
 
 	resp := provider.ImportResourceState(providers.ImportResourceStateRequest{
-		TypeName: n.Addr.Resource.Resource.Type,
-		ID:       n.ID,
-		ClientCapabilities: providers.ClientCapabilities{
-			DeferralAllowed:            false,
```

Comment:
Is it intentional that this attribute which used to be false, will now become true with this change?

Reply:
yes, we revisited the semantics of client capabilities. instead of "what can this request in this terraform version handle" it's rather "what can this terraform version handle", has been a problem in the initial implementation.